### DISCUSSION TITLE
Linux releases

### DISCUSSION THREAD

**Discussion by filip2cz** [brackets-cont/brackets#153] [Q&A] [answered]
When the Linux release comes out, will the original adobe flatpak and packages in the repositories be updated?

**Comment by abose** [comment, marked as answer]
Adobe never released a flatpack or published Brackets to any of the platform package managers in Linux as I remember. But going forward, we plan to do so.

Please track the Linux platform support in the related issue. closing as duplicate of https://github.com/brackets-cont/brackets/discussions/113

**Reply by filip2cz** [reply]
Idk, I have it in repositories on openSUSE and it is on Flathub (https://flathub.org/apps/details/io.brackets.Brackets), but they may be unofficial packages.
